src/components/Carousel/PrevNext.js

```diff
@@ -13,6 +13,7 @@ function PrevNext({ onClick, type }) {
         position: absolute;
         z-index: 1;
         top: 50%;
```

Comment:
I realize now that it wasn’t centered before:

```suggestion
         top: calc(50% - ${3 * GU}px);
```